codingTest_김준우/두 개 뽑아서 하기.js

```diff
@@ -0,0 +1,19 @@
+function solution(numbers) {
+  let Arr = [];
+  for (let i = 0; i < numbers.length - 1; i++) {
+    for (let j = 1; j < numbers.length; j++) {
+      if (j <= i) {
+        continue;
+      } else {
+        let indexJ = numbers[i] + numbers[j];
+        Arr.push(indexJ);
+      }
+    }
+  }
+  Arr = new Set(Arr);
+  Arr = [...Arr];
+  Arr = Arr.sort(function (a, b) {
+    return a - b;
+  });
```

Comment:
```suggestion
  arr = arr.sort((a, b) => {
    return a - b;
  });
```
익명함수는 이렇게 arrow function으로 사용이 가능합니다. 나중에는 이 방법을 더 많이 쓰니깐 알아두면 좋습니다.

Reply:
> 익명함수는 이렇게 arrow function으로 사용이 가능합니다. 나중에는 이 방법을 더 많이 쓰니깐 알아두면 좋습니다.
<img width="432" alt="스크린샷 2023-12-08 오후 9 04 07" src="https://github.com/modern-agile-team/7term-assignment/assets/137868493/51e172ef-592e-4381-994d-70904fa38663">

이 경우엔 중괄호랑 return도 생략해서 많이 쓰긴 합니다